examples/angular2/index.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html lang="en" data-framework="angular2">
+	<head>
+		<meta charset="utf-8">
+		<title>Angular2 • TodoMVC</title>
+		<link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
+		<link rel="stylesheet" href="node_modules/todomvc-common/base.css">
+		<script src="node_modules/bower-traceur-runtime/traceur-runtime.js"></script>
+		<script src="node_modules/systemjs/dist/system.js"></script>
+		<script src="node_modules/store.js/store.js"></script>
+		<script src="vendor/angular2.dev.js"></script>
+	</head>
+	<body>
+		<todo-app></todo-app>
+		<footer id="info">
+			<p>Double-click to edit a todo</p>
+			<p>
+				Created by <a href="http://github.com/samccone">Sam Saccone</a>
+				using <a href="http://angular.io">Angular2</a>
+			</p>
```

Comment:
Misses the ["part of todomvc"](https://github.com/tastejs/todomvc/blob/master/examples/angularjs_require/index.html#L61) paragraph